### ISSUE TITLE
Color of muted text fails contrast check

### ISSUE THREAD

**Issue by seanfagan** [seanfagan/seanfagan.github.io#8]
The `text-muted` color fails contrast checks. Brighten the color of muted text to at least #8a8a8a when on the dark background.
